src/js/common/schemaform/FormApp.jsx

```diff
@@ -31,6 +31,9 @@ class FormApp extends React.Component {
       newProps.router.push(newProps.returnUrl);
       // Set loadedStatus in redux to not-attempted to not show the loading page
       newProps.setFetchFormStatus(LOAD_STATUSES.notAttempted);
+    } else if (newProps.loadedStatus === LOAD_STATUSES.prefillComplete) {
+      newProps.router.push(newProps.routes[newProps.routes.length - 1].pageList[1].path);
```

Comment:
I learned this today, actually, but when you're in an component that has children that are routes, you get an array of routes, one for each nested child used by the current route. So it looks like ["/", "introduction"] when you're on the intro page (though with objects, not strings).